changelog/655.feature.rst

```diff
@@ -0,0 +1 @@
+Add support for subclassing ``.ext.tasks.Loop`` and using subclasses in :func:`.ext.tasks.loop` decorator
```

Comment:
```suggestion
|tasks| Add support for subclassing :class:`.ext.tasks.Loop` and using subclasses in :func:`.ext.tasks.loop` decorator
```

Reply:
Additionally, if the arguments are made into keyword arguments, that should be added as `655.breaking.rst`.